library/ajax/billing_tracker_ajax.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Ajax endpoint for interface/billing/billing_tracker.php,
+ * which is the interface that provides tracking information for a claim batch
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ken Chapple <ken@mi-squared.com>
+ * @copyright Copyright (c) 2021 Ken Chapple <ken@mi-squared.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
```

Comment:
if this only sends back a json, i wonder if it would be safer to set header for this:
```
header('Content-Type: application/json');
```
not sure though, any thoughts on this @sjpadgett , @stephenwaite ?

Reply:
Depends if calling request wants a string or object. If object, then use the header.
I don't remember the default dataType for ajax but i'd recommend using ES fetch where possible.

Reply:
It's for Jquery Datatables, not sure what the default request is.

Reply:
If you don't add a json datatype to the ajax request then jquery will figure it out. Looks to me like the ajax is already converting to an object to datatable so setting a header here doesn't gain anything imo. My preference is to transport json as a string anyway so, i'd leave as is....